glide.yaml

```diff
@@ -65,6 +65,8 @@ import:
   version: 3bc7a60b1df80a5766820c4a53d59668f4553f75
 - package: github.com/juju/errors
   version: c7d06af17c68cd34c835053720b21f6549d9b0ee
+- package: github.com/gbraad/go-hvkvp
+  version: a692e030379b6de28a10b7697decdf8109899f6e
```

Comment:
It would be better to use tag/release-version instead of sha.

Reply:
https://github.com/gbraad/go-hvkvp/releases/tag/0.3